pkg/server/events.go

```diff
@@ -118,13 +118,14 @@ func convertEvent(e *gogotypes.Any) (string, interface{}, error) {
 // start starts the event monitor which monitors and handles all container events. It returns
 // a channel for the caller to wait for the event monitor to stop. start must be called after
 // subscribe.
-func (em *eventMonitor) start() (<-chan struct{}, error) {
+func (em *eventMonitor) start() <-chan error {
```

Comment:
cool.. Should probably change the function declaration comments to reflect the change in return value. For example, "It returns an error return channel to the caller. Callers should wait for errors to be returned over the error return channel, such as Failed to handle event stream."

Reply:
Done.